examples/cmd/remote-snapshotter/README.md

```diff
@@ -0,0 +1,30 @@
+# Remote Snapshotter Example
+
+This example shows how to build a firecracker-containerd client that can launch lazily-loaded estargz images in a firecracker microvm. If you haven't already, please follow the [remote snapshotter getting started guide.](https://github.com/firecracker-microvm/firecracker-containerd/blob/main/docs/remote-snapshotter-getting-started.md)
+
+# Building
+
+The example can be built with
+
+```
+make remote-snapshotter
+```
+
+# Usage
+To use the example, run (e.g.)
+
+```
+./remote-snapshotter ghcr.io/stargz-containers/alpine:3.15.3-esgz
+```
+A list of pre-converted images for testing can be found in the [stargz-snapshotter documentation](https://github.com/containerd/stargz-snapshotter/blob/main/docs/pre-converted-images.md)
+
+
+This will launch the container inside a microVM with lazy loading supplied by the stargz-snaphotter and connect stdio to the container.
+
+Note: stargz-snapshotter has a fallback mechanism to use an embedded overlay snapshotter if an image is not in estargz format. This will not work with firecracker-containerd because the client is responsible for writing content into the snapshot, but the snapshot mountpoints are isolated inside the VM. 
+
+# Credentials
+
+By default, the example tool will ask for a docker username/password each time it is invoked. It can also read these from environment the variables `$DOCKER_USERNAME` and `$DOCKER_PASSWORD`. 
```

Comment:
IIRC, the example image above `ghcr.io/stargz-containers/alpine:3.15.3-esgz` requires a GitHub personal access token for pull.  I think we should mention and/or link to GitHub personal access token docs either in this section or Usage.